usecases/MinimumWorkingExample/dodo.py

```diff
@@ -374,3 +386,101 @@ def task_upload_to_openbis():
             'targets': [sample_file_path],
             'clean': [clean_targets],
         }
+
+@create_after(executed='extract_metadata_emodul')
+def task_perform_calibration():
+    """Loop over the experiments and store the calibrated E in csv file. Each iteration generates four files
+    viz. calibrated data.csv, displacement data used, load data used and the knowledge graph file containing the calibration details
+    (these are not specific file names)"""
+    # create folder, if it is not there
+    Path(calibrated_data_directory).mkdir(parents=True, exist_ok=True)
+
+    # defining calibration input, setting the values for the priors
+    E_loc = 30  # KN/mm2 (mean)
+    E_scale = 10  # KN/mm2 (std)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
+        list_exp_name = [single_example_name]
+    else:  # go through all files
+        list_exp_name = os.listdir(
+            processed_data_emodulus_directory)
+        list_exp_name = [os.path.splitext(f)[0] for f in list_exp_name]  # split extension
+    for f in list_exp_name:
+        calibrated_data_expwise = os.path.join(calibrated_data_directory,f) # create new folder for each exp
+        # create the folder if its not there
+        Path(calibrated_data_expwise).mkdir(parents=True,exist_ok=True)
+
+        # read in the metadata for each exp
+        exp_metadata_path = os.path.join(metadata_emodulus_directory, f + '.yaml')
+        with open(exp_metadata_path) as file:
+            data = yaml.safe_load(file)
+        diameter = float(data['diameter'])
+        #length = float(data['length'])
+        length = 100 # as suggested here :https://github.com/BAMresearch/LebeDigital/pull/152#discussion_r1187107430
```

Comment:
isn't the gauge length already stored in the metadata file?

Reply:
If it is, then this is a new addition, which I addressed recently. Not sure what the current state is. It is planned to be included and then it can be directly read and does not need to be hard coded anymore.

Reply:
Ok, then let's make an issue and as soon as this is integrated, we'll update that here.